### PR TITLE
Replaced one paper and updated page numbers as appropriate

### PR DESCRIPTION
**Pull request by paulratdollabs**
Replaced Thorisson paper with the latest version. Updated page numbers in followon papers.  Updated the bibteX file to reflect the changed page numbers.

### CODE REVIEW

File: IWSSL2020.bib [hunk 1]
```diff
@@ -143,7 +143,7 @@ @InProceedings{lieberman20
   title =	 {Intrinsic and Extrinsic Motivation in Intelligent
                   Systems},
   author =	 {Lieberman, Henry},
-  pages =	 {62--71},
+  pages =	 {71--79},
```
Comment (lawrennd) [inline IWSSL2020.bib]: Did this paper get shorter, shouldn't this be 80?

**Review by lawrennd — COMMENTED** [review]
Hi Paul, did one of the papers also get shorter? I notice a page got dropped for Lieberman in the total (but didn't double check the PDF).

File: IWSSL2020.bib (hunk 1 above)
Comment (paulratdollabs) [reply IWSSL2020.bib]: Sigh, yes, you are right.  I'll need to make the fixes and try again.

**Comment by paulratdollabs** [comment]
Thank you for pointing out that error in page numbering.  I have fixed it now and double checked it this time.

**Comment by paulratdollabs** [comment]
Let's try this pull request again.  page number errors have been fixed.

**Review by lawrennd — APPROVED** [review]
Looks good.

**Pull request merged**